src/tools/PICS-generator/PICSGenerator.py

```diff
@@ -47,6 +47,11 @@ def GenerateDevicePicsXmlFiles(clusterName, clusterPicsCode, featurePicsList, at
     nodeOperationalCredentialsCluster = "Node Operational Credentials Cluster"
     basicInformationCluster = "Basic Information Cluster"
     networkCommissioningCluster = "Network Commissioning Cluster"
+    diagnosticLogsCluster = "Diagnostic Logs Cluster"
+    thermostatUserInterfaceConfigurationCluster = "Thermostat User Interface Configuration Cluster"
+    wakeOnLANCluster = "Wake On LAN Cluster"
+    lowPowerCluster = "Low Power Cluster"
+    laundryModeCluster = "Laundry Mode Cluster"
```

Comment:
nah, I think this is ok - if you end up getting the newer name, it won't have to be changed. This will let the older stuff be used with your tool too. We should fix these naming issues for real, but for the purposes of this script, no harm leaving it.